onnx/backend/test/case/node/pool_op_common.py

```diff
@@ -27,10 +27,52 @@ def get_pad_shape(
         pass
     return pad_shape
 
+# Use pads to calculate output shape. Use output shape in turn to calculate the actual pads
```

Comment:
I would put that information in the docstring of the function rather than in a comment.

Reply:
good point. I added comment to the function body.